eng/common/scripts/get-codeowners.ps1

```diff
@@ -13,34 +16,89 @@ if (!(Test-Path $codeOwnersLocation)) {
 }
 
 $codeOwnersContent = Get-Content $codeOwnersLocation
+$previousLine
 
 foreach ($contentLine in $codeOwnersContent) {
   if (-not $contentLine.StartsWith("#") -and $contentLine){
     $splitLine = $contentLine -split "\s+"
     
     # CODEOWNERS file can also have labels present after the owner aliases
     # gh aliases start with @ in codeowners. don't pass on to API calls
-    $ownedFolders[$splitLine[0].ToLower().Trim("/")] = ($splitLine[1..$($splitLine.Length)] `
-      | ? { $_.StartsWith("@") } `
-      | % { return $_.substring(1) }) -join ","
+
+    $aliases = ($splitLine[1..$($splitLine.Length)] | ? { $_.StartsWith("@") } | % { return $_.substring(1) }) -join ","
+    $labels = ""
+
+    if ($null -ne $previousLine -and $previousLine.StartsWith("# PRLabel:"))
+    {
+      $previousLine = $previousLine.Replace("# PRLabel: ","")
+      $splitPrevLine = $previousLine -split "%" 
+      $labels = ($splitPrevLine[1..$($splitPrevLine.Length)] | % { return $_.Trim() }) -join ","
+    }
+
+    $ownedFolders[$splitLine[0].ToLower().Trim("/")] = @{ Aliases = $aliases; Labels = $labels }
   }
+  $previousLine = $contentLine
 }
 
 $results = $ownedFolders[$target]
 
 if ($results) {
-  Write-Host "Found a folder $results to match $target"
-  
-  if ($VsoVariable) {
-    $alreadyPresent = [System.Environment]::GetEnvironmentVariable($VsoVariable)
+  Write-Host "Found a folder to match $target"
+  $aliases = $results.Aliases -split ","
+  $users
+  $teams
+
+  if ($AuthToken) {
+    $headers = @{
+      Authorization = "bearer $AuthToken"
+    }
+  }
+
+  foreach ($str in $aliases)
+  {
+    $usersApiUrl = "https://api.github.com/users/$str"
+    try
+    {
+      $response = Invoke-RestMethod -Headers $headers $usersApiUrl
+      if ($users) { $users += ("," + $str) } else { $users += $str }
+    }
+    catch {
+      if ($_.Exception.Response.StatusCode.Value__ -eq 404) # Consider it a team Alias
```

Comment:
That doesn't seem like a save assumption. Why wouldn't we query for the teams under our org?

Reply:
What do plan to do with the teams?

Reply:
We were planing to auto tag PR with the team alias

Reply:
auto-tagging the PR's in the docs org will not work as the teams are not necessary the same in each org and the team cannot be accessed from another org.

Reply:
Am actually doing this to tag automatic version increment PRs.

Reply:
For automatic version increment PR's we shouldn't need to do any of this work. We already have the codeowners being added as reviewers by github and we have the labels being added by the area via the fabric bot. I thought you were doing this for doc PRs.

Reply:
Can you add a link to the issue you are trying to address in the PR description?

Reply:
I guess you are right. Looks like there is some confusion. This is the [original issue](https://github.com/Azure/azure-sdk-for-net/issues/14760) I set out to fix. But @scbedd requested reviews on my PRs ([python](https://github.com/Azure/azure-sdk-for-python/pull/13485), [java](https://github.com/Azure/azure-sdk-for-java/pull/14693), [net](https://github.com/Azure/azure-sdk-for-net/pull/14767)), he then opened [another issue](https://github.com/Azure/azure-sdk-tools/issues/954). I was attempting to get it all resolved.

Reply:
@chidozieononiwu OK so all that was really needed was a way to ignore teams then correct?  I'm fine with adding the other functionality to the script but I don't believe it is going to really be used anywhere yet.